docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -336,5 +336,12 @@ is adjusted to avoid a bit shift that is greater than the type length:
      ...
    }
 
-
-
+**How do I resolve missing include errors that occur when migrating my
+ code?
+
+If an error occurs because a file that was included could not be

```

Comment:
Good information! A couple suggestions: Order the information so that you first show how to output the verbose info (troubleshoot) and then to show how to add an extra include path (the solution). Can also simplify sentences a bit. For example: 

Use the option ``--extra-arg=-v`` to prompt |tool_name| to use verbose output, which includes information about which paths the tool searches for includes.

To add an additional path for the tool to use when searching for includes during migration, use the option ``--extra-arg="-I<extra include path>"`` in your migration command.